app/models/manageiq/providers/openstack/cloud_manager.rb

```diff
@@ -493,6 +493,10 @@ def self.description
     @description ||= "OpenStack".freeze
   end
 
+  def self.vm_vendor
+    @vm_vendor ||= "openstack".freeze
```

Comment:
out of curiosity, why the name `vm_vendor`?

Reply:
Because this is for the `vms#vendor` column and doesn't always match the provider "vendor"